dockerfiles/cfme_tests_base/Dockerfile

```diff
@@ -1,12 +0,0 @@
-# Our custom docker image with app
```

Comment:
These are the dockerfiles that are used for PRT, is the new one backwards compatible?